### ISSUE TITLE
Background color on RowSpan is missing color on last one.

### ISSUE THREAD

**Issue by mmDevel** [QuestPDF/QuestPDF#552]
**Describe the bug**

When I generate table with background for each of the cells and than I want to add background on "group of the cells,"
It will set background on all of the cells except the last one:
Without background in CellStyleMainTable
![image](https://user-images.githubusercontent.com/118260991/236825447-0a490fd1-6be6-4a00-b841-cad57296ffc1.png)
With background in CellStyleMainTable
![image](https://user-images.githubusercontent.com/118260991/236825563-b29a7210-11fb-4037-b8ae-b42539e940be.png)

You can even see that there is a border on left side of the last row, which should be included.
Workaround is to add background to whole table but if it is intended to use it this way, than it is a bug.

**To Reproduce**
Please provide an example, minimalistic code that shows the problem.

```
container.Table(table => {
            table.ColumnsDefinition(columns => {
                columns.RelativeColumn(0.3f);
                columns.RelativeColumn();
            });
            table.Header(header => {
                header.Cell().Element(CellStyle).Text("");
                header.Cell().Element(CellStyle).Text("man. 8.5");

                static IContainer CellStyle(IContainer container) {
                    return container.DefaultTextStyle(x => x.SemiBold()).AlignCenter();
                }
            });
            // Hour blocks
            for (int i = 6; i <= 20; i++) {
                // Minute blocks
                for (int j = 0; j < 2; j++) {
                    if (j == 0) {
                        table.Cell().Element(CellStyleMainTable).Text($"{i.ToString("00")}:00");
                    } else {
                        table.Cell().Element(CellStyleMainTable).Text("");
                    }
                    table.Cell().Element(CellStyleMainTable).Text("");
                }
            }
            static IContainer CellStyleMainTable(IContainer container) {
                 return container
                     .Border(0.5f).BorderColor(Colors.Blue.Lighten4)
                     //.Background(Colors.Blue.Lighten5)
                     .PaddingVertical(5);
             }

            table.Cell().Row(1).RowSpan(3).Column(1).Element(BlockAccepted).Text("3 rows");
            table.Cell().Row(1).RowSpan(6).Column(2).Element(BlockAccepted).Text("6 rows");


            static IContainer BlockAccepted(IContainer container) {
                return container
                    .Border(1.5f).BorderColor(Colors.Green.Lighten2)
                    .Background(Colors.Green.Lighten4);
            }
        });
```

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by mmDevel** [comment]
But it also seems that it is not happening with all of the colors, if the Background of the green blocks i set to medium than it is fine.

**Comment by MarcinZiabek** [comment]
Hello 😁

Thank you so much for finding and reporting this problem. It should be fixed in the `2022.12.6` and `2023.4.2` releases. Would you please test it?
